content/06.methods.md

```diff
@@ -316,7 +321,7 @@ We used the total number coding sequence consensus SNVs for the numerator and th
 We performed molecular subtyping on tumors in the OpenPBTA to the extent possible.
 The `molecular_subtype` field in `pbta-histologies.tsv` contains molecular subtypes for tumor types selected from `pathology_diagnosis` and `pathology_free_text_diagnosis` fields as described below, following World Health Organization 2016 classification criteria [@doi:10.1007/s00401-016-1545-1].
 
-Medulloblastoma (MB) subtypes SHH, WNT, Group 3, and Group 4 were predicted using the consensus of two RNA expression classifiers: Medulloblastoma Classifier [@doi:10.1371/journal.pcbi.1008263] and MM2S Classifier [@doi:10.1186/s13029-016-0053-y] on the RSEM FPKM data (`molecular-subtyping-MB` analysis module).
+Medulloblastoma (MB) subtypes SHH, WNT, Group 3, and Group 4 were predicted using the consensus of two RNA expression classifiers: `medullo-classifier-package` [@doi:10.1371/journal.pcbi.1008263] and `MM2S` [@doi:10.1186/s13029-016-0053-y] on the RSEM FPKM data (`molecular-subtyping-MB` analysis module).
```

Comment:
It looks like the package itself is named `MedulloClassifier` but the repo is `medullo-classifier-package`, so we should probably use the former here.